scripts/debug_tools/README.md

```diff
@@ -0,0 +1,32 @@
+These scripts may be used to aid debugging of clang crashes during analysis.
```

Comment:
Start the README with some sort of a header so that it looks like a proper markdown file:

```md
CTU Debugging
=============
```